HACKING.md

```diff
@@ -224,10 +224,57 @@ OpenShift is checked into this repository.  To install `godep` locally run:
 
 If you are not updating packages you should not need godep installed.
 
-## Cherry-picking an upstream commit into Origin
+## Cherry-picking an upstream commit into Origin: Why, how, and when.
+
+Upstream origin carries patches inside of vendor/ on top of each rebase.  Thus, Origin consumes Kubernetes in two ways.
```

Comment:
s/origin/Origin

I think you mean "Origin carries upstream patches" rather than "Upstream origin carries patches"